test/recording.ts

```diff
@@ -0,0 +1,57 @@
+import * as path from 'path';
+import { tmpdir } from 'os';
+import { mkdtemp, readdir } from 'fs-extra';
+import { focusChild, focusMain, test, useSpectron } from './helpers/spectron';
+import { setFormDropdown, setFormInput } from './helpers/spectron/forms';
+import { sleep } from './helpers/sleep';
+
+useSpectron({ appArgs: '--nosync' });
```

Comment:
'--nosync' is not necessary here, because this test doesn't use authorization

Reply:
Got it. Thanks.